app/models/mongo/UserDataTemplate.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.mongo
+
+trait UserDataTemplate {
+  val sessionId: String
+  val mtdItId: String
+  val nino: String
+  val taxYear: Int
```

Comment:
Let's change all the members to `def`

Reply:
I have amended this in the next commit

Reply:
Still need to be changed to `def`

Reply:
Thought I had amended this but must have just missed it, thanks Vincent for point this out. Have made the change in the latest commit.